### PR TITLE
Migrate to bpmn-js-properties-panel@1.0.0-alpha.0 and support Zeebe 1.3

### LINKED ISSUE #71: Publish (npm or branch/tag) a version with Zeebe 1.3 features and the new properties panel
__What should we do?__

Publish a version of camunda-bpmn-js with Zeebe 1.3 features and the new properties panel. This could be done via npm or a branch (like we have done so far).

__Why should we do it?__

To enable other teams (@CatalinaMoisuc and @ultraschuppi) to use the new properties panel with Zeebe 1.3 features.

### PR DESCRIPTION
**Pull request by barmac**
Closes #71

### CODE REVIEW

File: test/camunda-cloud/features/modeling/UpdatePropagateAllChildVariablesBehaviorIntegrationSpec.js [hunk 1]
```diff
@@ -347,6 +346,7 @@ describe('camunda-cloud/features/modeling - UpdatePropagateAllChildVariablesBeha
 // helper /////////
 
 const getPropagateAllChildVariablesToggle = (container) => {
+  debugger;
```
Comment (MaxTru) [inline test/camunda-cloud/features/modeling/UpdatePropagateAllChildVariablesBehaviorIntegrationSpec.js]: I think we dont want that

File: test/camunda-cloud/features/modeling/UpdatePropagateAllChildVariablesBehaviorIntegrationSpec.js [hunk 2]
```diff
@@ -23,23 +23,22 @@ import selectionModule from 'diagram-js/lib/features/selection';
...
 import zeebeModdleExtensions from 'zeebe-bpmn-moddle/resources/zeebe';
 
-import propertiesProviderModule from 'lib/camunda-cloud/features/properties-provider';
 import zeebeModelingModules from 'lib/camunda-cloud/features/modeling';
 
 import diagramXML from './process-call-activities.bpmn';
 
 
-describe('camunda-cloud/features/modeling - UpdatePropagateAllChildVariablesBehavior (integration tests)', function() {
+describe.only('camunda-cloud/features/modeling - UpdatePropagateAllChildVariablesBehavior (integration tests)', function() {
```
Comment (MaxTru) [inline test/camunda-cloud/features/modeling/UpdatePropagateAllChildVariablesBehaviorIntegrationSpec.js]: This does not belong here

**Review by MaxTru — CHANGES_REQUESTED** [review]
Some debugging artifacts were forgotten.

Apart from this: looks great!

File: test/camunda-cloud/features/modeling/UpdatePropagateAllChildVariablesBehaviorIntegrationSpec.js (hunk 2 above)
Comment (barmac) [reply test/camunda-cloud/features/modeling/UpdatePropagateAllChildVariablesBehaviorIntegrationSpec.js]: I am wondering how it was ignored by the linter 🤯

Comment (barmac) [reply test/camunda-cloud/features/modeling/UpdatePropagateAllChildVariablesBehaviorIntegrationSpec.js]: all right, I added this and then removed :/

File: test/camunda-cloud/features/modeling/UpdatePropagateAllChildVariablesBehaviorIntegrationSpec.js (hunk 1 above)
Comment (barmac) [reply test/camunda-cloud/features/modeling/UpdatePropagateAllChildVariablesBehaviorIntegrationSpec.js]: Fixed.

**Review by MaxTru — APPROVED** [review]
Great success!

**Pull request merged**